contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -1166,18 +1408,37 @@ def read_from_file(filepath, simplify=True, transcribe_log=False):
 
         storage = aaf_file.content
 
+        # Get just the top-level MOBS from the AAF
+        topmobs = list(storage.toplevel())
+
         # Note: We're skipping: f.header
         # Is there something valuable in there?
-        _transcribe_log("---\nTranscribing top level mobs\n---", 0)
 
-        # Get just the top-level MOBS from the AAF
-        top = list(storage.toplevel())
+        if topmobs:
+            _transcribe_log(
+                "---\nTranscribing {} top level mobs\n---".format(
+                    len(topmobs)
+                ),
+                0
+            )
+            # Transcribe just the top-level mobs
+            result = _transcribe(topmobs, parents=list(), edit_rate=None)
+        else:
+            mastermobs = list(storage.mastermobs())
+            _transcribe_log(
+                "---\nTranscribing {} master mobs "
+                "(no top level composition mobs found)\n---".format(
+                    len(mastermobs)
+                ),
+                0
+            )
+            result = _transcribe(mastermobs, parents=list(), edit_rate=None)
```

Comment:
TODO: Let's check to see if this is left over from a previous merge/change. It seems unrelated to the marker feature.

Reply:
Yes, this looks like a reversion of some code due to this PR changing hands several times. In video-conference code review with @timlehr he says he can fix this.

Reply:
Hey Josh, I was off on vacation, but I will look into this soon.

Reply:
Thanks @timlehr - let us know if we can help.